src/components/pages/ViewAllMeetings/MeetingInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MeetingInfo = props => {
+  const { info } = props;
+
+  return (
+    <div>
+      <p>{info.meeting_id}</p>
+      <p>{info.created_at}</p>
+      <p>{info.updated_at}</p>
+      <p>{info.meeting_topic}</p>
+      <p>{info.meeting_time}</p>
+      <p>{info.host_id}</p>
```

Comment:
I'm not sure if the user would need to see the host_id and attendee_id. Maybe just make it so it shows the name of the attendee and host(probably need some back-end work for that, i'm glad to jump in).